### PR TITLE
RIA-4104 enable callback ccdSubmitted endpoint to ia-case-notificatio…

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###
[RIA-4104](https://tools.hmcts.net/jira/browse/RIA-4104)

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RWA-318-Allocate-a-case-to-caseworker-midevent.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "description": "RWA-318 Allocate a case to caseworker midEvent",
-  "disabled": "false",
+  "disabled": "true",
```
Comment (makowalski) [inline src/functionalTest/resources/scenarios/RWA-318-Allocate-a-case-to-caseworker-midevent.json:3]: RWA tests needs to work before merge to the master, we should not disable them

File: src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario-midevent.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "description": "RWA-340 Allocate a case to caseworker when there is no assignments for the location midEvent",
+  "disabled": "true",
```
Comment (makowalski) [inline src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario-midevent.json:3]: same here

File: src/functionalTest/resources/scenarios/RWA-318-Allocate-a-case-to-caseworker-midevent.json (hunk 1 above)
Comment (mkayad) [reply src/functionalTest/resources/scenarios/RWA-318-Allocate-a-case-to-caseworker-midevent.json]: i reset this test on master and it is not part of my PR anymore

File: src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario-midevent.json (hunk 2 above)
Comment (mkayad) [reply src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario-midevent.json]: i reset this test on master and it is not part of my PR anymore

**Review by makowalski — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**